pangeo_forge_recipes/storage.py

```diff
@@ -25,11 +25,15 @@ def _get_url_size(fname, **open_kwargs):
     return size
 
 
-def _copy_btw_filesystems(input_opener, output_opener, BLOCK_SIZE=10_000_000):
+def _copy_btw_filesystems(input_opener, output_opener, BLOCK_SIZE=10_000_000, **kwargs):
+    streaming = True if kwargs.pop("block_size", True) == 0 else False
```

Comment:
Would prefer an explicit keyword argument rather than generic `**kwargs`.

Reply:
Also find it confusing to have both bases `BLOCK_SIZE` and `block_size` appearing in the arguments. Are these treated differently?

Reply:
> Would prefer an explicit keyword argument rather than generic **kwargs.

Makes sense; that's an easy fix.

> Also find it confusing to have both bases BLOCK_SIZE and block_size appearing in the arguments. Are these treated differently?

Yeah, agree that's not ideal. Here, `BLOCK_SIZE` is the actual block size to read (as it's always been), and `block_size` is the keyword arg passed to `fsspec_open_kwargs` to trigger streaming reads.